ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.txt

```diff
@@ -130,14 +130,14 @@ hideall "--Reset--"
 
 
 ### Phase 4: Snacks ###
-800 "Freefire" sync /:Titan:2CF5:/ window 800,0
+800 "Freefire" sync /:The Ultima Weapon:2CF5:/ window 800,0
```

Comment:
I think you already fixed this in #1611.  Did you update your branch locally with the latest changes?  (Can I help?)